crates/sui-indexer/src/models/objects.rs

```diff
@@ -79,6 +79,9 @@ pub struct StoredObjectSnapshot {
     pub owner_type: Option<i16>,
     pub owner_id: Option<Vec<u8>>,
     pub object_type: Option<String>,
+    pub object_type_package: Option<Vec<u8>>,
```

Comment:
I'm surprised these fields were not in `StoredObjectSnapshot` and things worked fine. I guess before the compatibility check we never had to use `StoredObjectSnapshot` in graphql.

Reply:
This is because we use the `StoredHistoryObject` type for both `objects_history` and `objects_snapshot`  haha...